aptos-move/block-executor/src/executor.rs

```diff
@@ -690,7 +701,7 @@ where
                     approx_output_size,
                 );
 
-                if accumulator.should_end_block_parallel() {
+                if txn_idx == scheduler.num_txns() - 1 && accumulator.should_end_block_parallel() {
                     // Set the execution output status to be SkipRest, to skip the rest of the txns.
                     last_input_output.update_to_skip_rest(txn_idx);
```

Comment:
`txn_idx` is the txn that exceeds limit? Shouldn't we set `txn_idx+1` to skip?

Reply:
I think SkipRest means skip after this one

Reply:
Ah correct